spec/models/user_spec.rb

```diff
@@ -1,5 +1,14 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "can have many songs" do
+    room = Room.create!(name: "Hatch")
+    user = User.create!(name: "Dan", room: room)
+    song1 = Song.create!(name: "Whats my age again?", room: room)
+    song2 = Song.create!(name: "Damnit", room: room)
+    user.songs << song1
+    user.songs << song2
+    user.save!
+    expect(user.songs).to match_array([song1, song2])
```

Comment:
best practice with ActiveRecord in specs like this is to explicitly `.reload` the model before assertion.

so `expect(user.reload.songs).to match_array([song1, song2])` is a little more bulletproof.

#nonblocker